pkg/config/config.go

```diff
@@ -302,7 +304,8 @@ func InitConfig(config Config) {
 	// Forwarder
 	config.BindEnvAndSetDefault("additional_endpoints", map[string][]string{})
 	config.BindEnvAndSetDefault("forwarder_timeout", 20)
-	config.BindEnvAndSetDefault("forwarder_retry_queue_max_size", 30)
+	config.BindEnvAndSetDefault("forwarder_retry_queue_max_size", DefaultForwarderRetryQueueMaxSize)
```

Comment:
Why not set this to -1 by default ? That way, if it' not -1, it was explicitly set by the user and we use it + a warning. If it's -1 we ignore it and use `forwarder_retry_queue_payloads_max_size`.